src/components/earn/PoolCardTri.styles.tsx

```diff
@@ -4,9 +4,13 @@ import { lighten, darken } from 'polished'
 import { TYPE } from '../../theme'
 import Card from '../Card'
 import { ButtonPrimary } from '../Button'
-import { ExternalLink } from '../../theme'
 
-export const Wrapper = styled(Card)<{ bgColor1: string | null; bgColor2?: string | null; isDoubleRewards: boolean }>`
+export const Wrapper = styled(Card)<{
+  bgColor1: string | null
+  bgColor2?: string | null
```

Comment:
Does `bgColor2` need to be optional _and_ nullable?

```suggestion
  bgColor2?: string
```